test/reviewers/sd/CardinalityReviewer.test.ts

```diff
@@ -128,71 +164,71 @@ describe('CardinalityReviewer', () => {
   });
 
   describe('when A cardinality is 0..1', () => {
-    it('should assess as equivalent to B cardinality 0..*', () => {
+    it('should assess as subset of B cardinality 0..*', () => {
       expectResult(0, '1', 0, '*', ReviewResult.SUBSET);
     });
 
-    it('should assess as superset of B cardinality 0..1', () => {
+    it('should assess as equivalent to B cardinality 0..1', () => {
       expectResult(0, '1', 0, '1', ReviewResult.EQUIVALENT);
     });
 
     it('should assess as superset of B cardinality 0..0', () => {
       expectResult(0, '1', 0, '0', ReviewResult.SUPERSET);
     });
 
-    it('should assess as superset of B cardinality 1..*', () => {
+    it('should assess as overlapping with B cardinality 1..*', () => {
       expectResult(0, '1', 1, '*', ReviewResult.OVERLAPPING);
     });
 
     it('should assess as superset of B cardinality 1..1', () => {
       expectResult(0, '1', 1, '1', ReviewResult.SUPERSET);
     });
 
-    it('should assess as superset of B cardinality 1..1', () => {
+    it('should assess as disjoint with B cardinality 1..1', () => {
       expectResult(0, '1', 2, '3', ReviewResult.DISJOINT);
     });
   });
 
   describe('when A cardinality is 0..0', () => {
-    it('should assess as equivalent to B cardinality 0..*', () => {
+    it('should assess as subset of B cardinality 0..*', () => {
       expectResult(0, '0', 0, '*', ReviewResult.SUBSET);
     });
 
-    it('should assess as superset of B cardinality 0..1', () => {
+    it('should assess as subset of B cardinality 0..1', () => {
       expectResult(0, '0', 0, '1', ReviewResult.SUBSET);
     });
 
-    it('should assess as superset of B cardinality 0..0', () => {
+    it('should assess as equivalent to B cardinality 0..0', () => {
       expectResult(0, '0', 0, '0', ReviewResult.EQUIVALENT);
     });
 
-    it('should assess as superset of B cardinality 1..*', () => {
+    it('should assess as disjoint with B cardinality 1..*', () => {
       expectResult(0, '0', 1, '*', ReviewResult.DISJOINT);
     });
 
-    it('should assess as superset of B cardinality 1..1', () => {
+    it('should assess as v B cardinality 1..1', () => {
```

Comment:
One of the pastes didn't go through correctly:
```suggestion
    it('should assess as disjoint with B cardinality 1..1', () => {
```

Reply:
Oh man.  This is embarrassing.  Once again, I need an AI-enabled VS Code plugin to do this all for me.

What's worse is that I _thought_ I double-checked this all yesterday before pushing.  I guess I should have triple-checked.

Reply:
All the magic is in 4c465c6.  Although I just noticed I could have easily just committed your suggestion.

I must be off my game today.